apps/studio/state/app-state.ts

```diff
@@ -51,9 +47,70 @@ const INITIAL_AI_ASSISTANT: AiAssistantPanelType = {
   tables: [],
 }
 
+const EMPTY_DASHBOARD_HISTORY: DashboardHistoryType = {
+  sql: undefined,
+  editor: undefined,
+}
+
+const getInitialState = () => {
+  if (typeof window === 'undefined') {
+    return {
+      aiAssistantPanel: INITIAL_AI_ASSISTANT,
+      dashboardHistory: EMPTY_DASHBOARD_HISTORY,
+      activeDocsSection: ['introduction'],
+      docsLanguage: 'js',
+      showProjectApiDocs: false,
+      isOptedInTelemetry: false,
+      showEnableBranchingModal: false,
+      showFeaturePreviewModal: false,
+      selectedFeaturePreview: '',
+      showAiSettingsModal: false,
+      showGenerateSqlModal: false,
+      navigationPanelOpen: false,
+      navigationPanelJustClosed: false,
+    }
+  }
+
+  const stored = localStorage.getItem(LOCAL_STORAGE_KEYS.AI_ASSISTANT_STATE)
+
+  const urlParams = new URLSearchParams(window.location.search)
+  const aiAssistantPanelOpenParam = urlParams.get('aiAssistantPanelOpen')
+
+  const parsedAiAssistant = stored
+    ? JSON.parse(stored, (key, value) => {
+        if (key === 'createdAt' && value) {
+          return new Date(value)
+        }
+        return value
+      })
+    : INITIAL_AI_ASSISTANT
```

Comment:
There's no reason that this value should be improperly formatted, but since it's user controlled we should properly err on the side of caution and wrap it with a try/catch?